workspaces/global-header/plugins/global-header/src/components/HeaderIconButtonComponent/HeaderIconButton.tsx

```diff
@@ -35,15 +38,37 @@ export const HeaderIconButton = ({
   tooltip,
   to,
 }: HeaderIconButtonProps) => {
+  const showNotifications = icon === 'notifications';
+  const unreadCount = useNotificationCount(showNotifications);
+  const isExternal = to && isExternalUrl(to);
+  const buttonProps: Record<string, any> = {
+    component: isExternal ? 'a' : Link,
+  };
+
+  if (to) {
+    buttonProps[isExternal ? 'href' : 'to'] = to;
+  }
+
+  if (isExternal) {
+    buttonProps.target = '_blank';
+    buttonProps.rel = 'noopener noreferrer';
+  }
+
   return (
     <Tooltip title={tooltip ?? icon}>
       <IconButton
         color="inherit"
-        aria-label="help"
+        aria-label={icon}
```

Comment:
I'm worried that the icon string isn't valuable for the user as an "alternative".

It's a good point. Maybe each icon should have a `label`  or `ariaLabel` as well that is used here?

For now I guess the tooltip is more useful then the icon. Wdyt?